src/index.js

```diff
@@ -1,6 +1,21 @@
+const Graceful = require('node-graceful');
 const { backup } = require('./backup');
 const { restore } = require('./restore');
 
+Graceful.timeout = 5000;
+Graceful.exitOnDouble = true;
+Graceful.captureExceptions = false;
+
+Graceful.on('exit', signal => {
+  console.log(`Received ${signal} signal, exiting process ...`);
+  process.exit(0);
+});
+
+process.on('uncaughtException', err => {
```

Comment:
This should be already handled by `Graceful.captureExceptions`

Reply:
Removed, thanks!